game/game.js

```diff
@@ -42,26 +41,30 @@ function disableBoard () {
 
 function updateScore () {
   document.querySelector('.score-x').textContent = score.x
-  document.querySelector('.score-y').textContent = score.y
+  document.querySelector('.score-o').textContent = score.o
 }
 
 function tictactoe (event) {
   var tile = event.target
+  var currentPlayer = document.body.className.toUpperCase()
   if (!tile.className.includes('tile')) return
   if (tile.textContent) return
-  tile.textContent = player ? 'X' : 'O'
+  tile.textContent = currentPlayer
   tile.removeEventListener('click', tictactoe)
   tile.classList.remove('active')
-  winner = findWinner()
+  var winner = findWinner()
   if (winner) {
-    update.textContent = winner + ' wins!'
-    if (winner === 'X') {
+    update.textContent = currentPlayer + ' wins!'
+    if (currentPlayer === 'X') {
       score.x += 1
       player = false
     } else {
-      score.y += 1
+      score.o += 1
       player = true
     }
+    var tilesOff = tiles.filter((tile, index) => winner.indexOf(index) < 0)
+    tilesOff.forEach(tileOff => tileOff.classList.add('tile-off'))
```

Comment:
Chain this instead of creating a throwaway variable.
```js
tiles
  .filter((tile, index) => winner.indexOf(index) < 0)
  .forEach(tileOff => tileOff.classList.add('tile-off'))
```